<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest_Graph">
    <property name="title">
     <string>Test Graph</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionToggle_force_calculation"/>
    <addaction name="actionToggle_rendering"/>
    <addaction name="actionShow_graph_size"/>
    <addaction name="actionExpand_by_generation"/>
   </widget>
   <addaction name="menuTest_Graph"/>
  </widget>
  <action name="actionAdd_Node">
   <property name="text">
    <string>add Node</string>
   </property>
  </action>
  <action name="actionToggle_force_calculation">
   <property name="text">
    <string>toggle force calculation</string>
   </property>
  </action>
  <action name="actionShow_graph_size">
   <property name="text">
    <string>show graph size</string>
   </property>
  </action>
  <action name="actionExpand_by_generation">
   <property name="text">
    <string>expand by generation</string>
   </property>
  </action>
  <action name="actionToggle_rendering">
   <property name="text">
    <string>toggle rendering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
